glide.yaml

```diff
@@ -13,7 +13,7 @@ import:
 - package: github.com/sirupsen/logrus
   version: ^0.10.0
 - package: github.com/docker/docker
-  version: d032264e1365a1cea621b7105e1ca521cb4218ef
+  version: v17.03.2-ce
```

Comment:
seems like @tmjd removed this and added that git hash. @tmjd any reason why you had to pin to that specific hash?

Reply:
That was the version that I found would work. All the tagged versions of docker/docker pulled in `Sirupsen/logrus` which conflict with the lowercased version.

Reply:
Hey @tmjd - thanks for the comment.

`v17.03.2-ce` works with logrus, not sure why we were seeing that error. It also fixes our ability to test against updated busybox containers.

@gunjan5 - any other questions?